roles/satellite-clone/tasks/restore_mongo_dump.yml

```diff
@@ -1,5 +1,5 @@
 - name: Drop old mongo database
-  command: echo 'db.dropDatabase();' | mongo pulp_database
+  command: mongo pulp_database --eval "db.dropDatabase()"
```

Comment:
I see a similar call in satellite-clone/tasks/reset_pulp_data.yml, should it also be updated?

Reply:
Thanks brad, I updated that line as well, its possible that that wasn't executing properly as well so good catch.

Since that runs only on backups w/o pulp_data I'll add that scenario to jenkins and re-run.